### PR TITLE
Feature/cart

### PR DESCRIPTION
**Pull request by Ochegbinu**

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -41,3 +42,4 @@
 Route::post('/product/create', [ProductController::class, 'store'])->name('storeProduct');
 
 
+Route::get('/add/cart/{cart}', [CartController::class, 'show'])->name('cart');
```
Comment (angwa) [inline routes/web.php]: Is the route endpoint correct Like this? I think it should be something like `cart/add/{cart} `

File: app/Http/Controllers/HomeController.php [hunk 2]
```diff
@@ -29,7 +30,9 @@ public function index()
 
     public function create()
     {
+        $cart = Cart::all();
         $product = Product::all();
-        return view('welcome', compact('product'));
+        $counters = Cart::all('product_id')->count();
+        return view('welcome', compact('product', 'cart', 'counters'));
```
Comment (angwa) [inline app/Http/Controllers/HomeController.php:39]: Always give your code spaces, especially before you return anything... Like where you are returning to view

**Review by angwa — CHANGES_REQUESTED** [review]

File: routes/web.php [hunk 3]
```diff
@@ -41,3 +42,5 @@
 Route::post('/product/create', [ProductController::class, 'store'])->name('storeProduct');
 
 
+Route::get('/add/cart/{id}', [CartController::class, 'show'])->name('cart');
+Route::get('/my/cart', [CartController::class, 'create'])->name('myCart');
```
Comment (angwa) [inline routes/web.php]: This is not correct routing na... you can use cart/show.. what is my/cart? Does it sense to your ear?

**Review by angwa — CHANGES_REQUESTED** [review]

File: routes/web.php (hunk 1 above)
Comment (Ochegbinu) [reply routes/web.php]: corrected

File: app/Http/Controllers/HomeController.php (hunk 2 above)
Comment (Ochegbinu) [reply app/Http/Controllers/HomeController.php:39]: owk sir, Noted!

File: routes/web.php (hunk 3 above)
Comment (Ochegbinu) [reply routes/web.php]: Corrected Sir. Noted!

**Pull request merged**